app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -121,55 +180,225 @@ def _create_task_description(
+
+            # Prepare system message with instructions
+            system_message = self._create_multimodal_system_message(ctx)
+
+            # Prepare conversation history (text only for now)
+            messages = [{"role": "system", "content": system_message}]
+
+            # Add conversation history
+            for msg in ctx.history[-6:]:  # Limit history to avoid token overflow
+                if isinstance(msg, str):
+                    # Parse the message format: "MessageType: content"
+                    if ": " in msg:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)